.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
     secure: RGovyUnMw3fp/bHZi058JvANT1rYmNqrsuSYew0cIgirO6YbMHr/rsjwCm1FTYpBl8s1zgr+u2b8ftYnfnCz2YT+Aip4NWrVYpVU0FEmfytGILrnUS0pjlt8m7fU9AKR1ElOSll7yw7e1kftynN39Q321etvwbLZcXon6zz0suE=
 
 matrix:
+    fast_finish: true
```

Comment:
This isn't a good change. You want to be able to see problems for each OS or debug/release combination. They are not always the same. They tend to be different more often than not. If you stop on first break. Then you won't catch the other different problems.